examples/dreambooth/train_dreambooth_flax.py

```diff
@@ -668,7 +649,12 @@ def checkpoint(step=None):
 
         if args.push_to_hub:
             message = f"checkpoint-{step}" if step is not None else "End of training"
-            repo.push_to_hub(commit_message=message, blocking=False, auto_lfs_prune=True)
+            upload_folder(
+                repo_id=repo_id,
+                folder_path=args.output_dir,
+                commit_message=message,
+                ignore_patterns=["step_*", "epoch_*"],
+            )
```

Comment:
I think it's okay as is because this is what we follow for the rest of the examples:

https://github.com/huggingface/diffusers/blob/57239dacd03c55ab6913efe0bf6ef7bb958f3c92/examples/dreambooth/train_dreambooth_lora.py#L1422

See here for additional reference:
https://github.com/huggingface/diffusers/pull/2934